### ISSUE TITLE
DocumentCard A11y Pass

### ISSUE THREAD

**Issue by dollykumari5** [OfficeDev/office-ui-fabric-react#10646]
Tracking all of the issues brought up with the 09/23 accessibility pass on Divider control
Details on individual bugs can be found on [Sharepoint](https://microsoft-my.sharepoint.com/:x:/p/v-amaver/ESqgICSURf9JoWoQf78p3zYBEqG1wJUIivILN0qRQCiPuQ?e=uAlVa5)

- Bug 1: [Accessibility][Visual Requirements]: "Marketing Documents" link cannot be identified as link since it appears as normal text. (#10659)
- Bug 2: [Accessibility][Keyboard Navigations]: Dotted visual focus indicator is not properly visible on document cards. (#10661)
- Bug 3: [Accessibility][Supporting the Platform]:In high Contrast mode black and white, focus indicator on links are not properly visible. (#10665)
- Bug 4:  [Suggestion][Usable][Accessibility]: Documentcard should be defined inside the landmark with brief aria-label. (#10664)

**Comment by dzearing** [comment]
@khmakoto Could you please triage these issues, open individual tracking bugs for actionable things, and then close this one?

**Comment by khmakoto** [comment]
@dzearing Sure, I'll do that.

**Comment by khmakoto** [comment]
All the bugs above have been split into their own corresponding issues and linked above, so I'll close this issue for now.
